libraries/fabric-shim/types/index.d.ts

```diff
@@ -53,7 +53,7 @@ declare module 'fabric-shim' {
         start(): Promise<void>;
     }
 
-    export interface ChaincodeServerOpts {
+    export interface ChaincodeServerOpts extends Partial<GRPCOptions> {
```

Comment:
It is annoying that the grpc-js GRPCOptions also contains the index signature `[key: string]: any`, which makes our type extending it less helpful for users since it won't flag up invalid or typos in property names as an error.

How about this as an approach?

```suggestion
    type GRPCOptions = {
        [K in keyof ChannelOptions as string extends K ? never : K]?: ChannelOptions[K];
    }
    
    export interface ChaincodeServerOpts extends GRPCOptions {
```

I think it more accurately reflects the code behaviour, where only property names starting with `grpc.` are considered.